cloudify_async_client/client.py

```diff
@@ -25,10 +25,19 @@ def url(self):
         return '{0}://{1}:{2}/api/{3}'.format(self.protocol, self.host,
                                               self.port, self.api_version)
 
-    def get(self, url, **kwargs):
+    def get(self, url, params=None, **kwargs):
         timeout = kwargs.pop('timeout', 300)
+        if timeout is None:
+            timeout = 300
         if isinstance(timeout, int) or isinstance(timeout, float):
             timeout = aiohttp.ClientTimeout(total=timeout)
```

Comment:
Also, maybe just call aiohttp.ClientTimeout on it here- I'm assuming that we don't really support timeout being anything that isn't an integer or a float (do we?)

Reply:
I'd like to make it still possible to pass `aiohttp.ClientTimeout` as a `timeout` parameter (in case someone wants to fine-tune the specific timeouts).

```suggestion
 def get(self, url, params=None, timeout=300, **kwargs):
        if isinstance(timeout, int) or isinstance(timeout, float):
            timeout = aiohttp.ClientTimeout(total=timeout)
```

Reply:
Ah, I see your reasoning- yeah, makes sense.